newChecklist.md

```diff
@@ -1,155 +0,0 @@
-1. [CODE STYLE] - Don't use one-letter variable naming. (It could be confusing for another developer, who reads your code. Also, it would really hard to find this variable in the code editor using search)
```

Comment:
Consider using more descriptive and concise language in the checklist items. For example, instead of 'Don't use one-letter variable naming', consider 'Use descriptive variable names'. 🤔